assets/prototype/infrastructure/services/contextProviders/ConfigProvider.js

```diff
@@ -41,7 +42,7 @@ const ConfigProvider = ({ children }) => {
 		],
 		currentUser
 	);
-	const value = { ...generalSettingsProps, ...currentUserProps };
+	const value = { dateFormat, timeFormat, ...currentUserProps };
```

Comment:
same thing here... do we need `dateTimeFormat` too ?

Reply:
no this is the data we get from `GET_GENERAL_SETTINGS` query, WPGraphQL isn't exposing `dateTimeFormat`:

![image](https://user-images.githubusercontent.com/1477453/70611471-edeb6100-1c0d-11ea-8143-c625b8d34ff0.png)